bigquery_etl/cli/backfill.py

```diff
@@ -355,40 +350,46 @@ def process(ctx, qualified_table_name, sql_dir, project_id, dry_run):
     if backfills_to_process_dict:
         entry_to_process = backfills_to_process_dict[qualified_table_name]
 
-        backfill_staging_qualified_table_name = (
-            get_backfill_staging_qualified_table_name(
-                qualified_table_name, entry_to_process.entry_date
-            )
-        )
+        click.echo(f"\nProcessing backfills dry runs for {qualified_table_name}:")
```

Comment:
nit or similar wording:
```suggestion
        click.echo(f"\nValidating backfill for {qualified_table_name} via dry run:")
```